templates/threatconnect.hbs

```diff
@@ -982,56 +1088,183 @@
+                            {{on "input" (fn (action "updateDescription" case.id))}}
+                          >
+                            {{or case.__newDescription case.description}}
+                          </textarea>
+                        {{else}}
+                          <span class="info-bar-value">{{case.description}}</span>
+                        {{/if}}
+                      </div>
+                      <div class="info-bar-row">
+                        <div class="info-bar-pill">
+                          <span class="info-bar-key">Status</span>
+                          {{#if case.__isEditing}}
+                            <select
+                              class="info-bar-value"
+                              value={{or case.__newStatus case.status}}
+                              {{on "change" (fn (action "updateStatus" case.id))}}
+                            >
+                              <option value="Open">Open</option>
+                              <option value="Closed">Closed</option>
+                            </select>
+                          {{else}}
+                            <span class="info-bar-value">{{case.status}}</span>
+                          {{/if}}
                         </div>
-                        <div class='info-bar-pill'>
-                          <span class='info-bar-key'>Severity</span>
-                          <span class='info-bar-value {{case.__severityColor}}'>
-                            {{case.severity}}
-                          </span>
+
+                        <div class="info-bar-pill">
+                          <span class="info-bar-key">Severity</span>
+                          {{#if case.__isEditing}}
+                            <select
+                              class="info-bar-value"
+                              value={{or case.__newSeverity case.severity}}
+                              {{on "change" (fn (action "updateSeverity" case.id))}}
+                            >
+                              <option value="Critical">Critical</option>
+                              <option value="High">High</option>
+                              <option value="Medium">Medium</option>
+                              <option value="Low">Low</option>
+                            </select>
+                          {{else}}
+                            <span class="info-bar-value">{{case.severity}}</span>
+                          {{/if}}
                         </div>
-                        <div class='info-bar-pill'>
-                          <span class='info-bar-key'>Resolution</span>
-                          <span class='info-bar-value'>
-                            {{case.resolution}}
-                          </span>
+
+                        <div class="info-bar-pill">
+                          <span class="info-bar-key">Resolution</span>
+                          {{#if case.__isEditing}}
+                            <select
```

Comment:
For all of these form elements they should be disabled when we're saving the edits

```
disabled={{case.__isSaving}}
```

(just like how we want to disable the "save" and "cancel" button once the user clicks on it)

Note that you might need to add the "__isSaving" property.